examples/krb5conn.py

```diff
@@ -0,0 +1,18 @@
+import stardog
+from requests_kerberos import HTTPKerberosAuth
+
+
+def main():
+    connection_details = {
+        'endpoint': 'http://localhost:55820',
```

Comment:
Should this be port 5820 or maybe just left out since endpoint defaults to `http://localhost:5820`?

Reply:
I was using that port when testing but I should change it.